platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/attribution/Attribution.java

```diff
@@ -8,7 +8,7 @@
   private static final String OPENSTREETMAP_ABBR = "OSM";
   static final String TELEMETRY = "Telemetry Settings";
 
-  static final String IMPROVE_MAP_URL = "https://www.mapbox.com/feedback/";
+  static final String IMPROVE_MAP_URL = "https://www.mapbox.com/map-feedback/";
```

Comment:
This might override https://github.com/mapbox/mapbox-gl-native/pull/13710/files#diff-67ba6ea072b572da2797c4c860f7fa05R11.